h2o-r/tests/testdir_munging/runit_distance.R

```diff
@@ -0,0 +1,38 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+test.dist <- function() {
+  iris_h2o <- as.h2o(iris)
+
```

Comment:
Probably safer to read in iris dataset.  Here we are relying on R to always have it.